### DISCUSSION TITLE
Do not merge notifications if there are only few files imported

### DISCUSSION THREAD

**Discussion by agatemosu** [ppy/osu#22631] [Ideas]
You could put a minimum number of imported files so that the notifications are merged.
If you import two it seems too few notifications for them to be merged.
They are merged when there are at least two imported files.

**Comment by peppy** [comment]
How are you exporting files?

**Reply by agatemosu** [reply]
When you export maps or skins by dragging from the explorer to osu!

**Reply by peppy** [reply]
Right, I think you mean import, not export.

We could consider that.

An alternative way might be to show more details of the import process inside the notification (maybe the last x items imported) so you can click on them individually?

**Reply by agatemosu** [reply]
yes haha ​​I got confused

could be like osu-web notifications
![image](https://user-images.githubusercontent.com/58957468/218706610-e2730cff-ce0d-4ab3-8f08-6c48e090ac5d.png)
